src/items/unions.md

```diff
@@ -66,14 +66,18 @@ unsafe {
 }
 ```
 
-Writes to `Copy` union fields do not require reads for running destructors, so
-these writes don't have to be placed in `unsafe` blocks
+Writes to [`Copy`] or [`ManuallyDrop`][ManuallyDrop] union fields do not
+require reads for running destructors, so these writes don't have to be placed
+in `unsafe` blocks
```

Comment:
The actual implementation goes a bit further, so `u.f1.g2 = ...` is safe if *that assignment* is at `ManuallyDrop` or `impl Copy` type... but on stable code that is impossible to observe since all fields must be `ManuallyDrop` or `impl Copy`. Not if the reference should even mention it, then.

Reply:
I think it's fine to leave out for now.  Is it even possible on nightly to have a non-Copy/ManuallyDrop type?

Reply:
Yeah, with a feature flag you can use any type without drop glue.

In principle even those could be safe to assign, so we could remove this safety check entirely... for now I went with the smaller change that should just make the safety check unreachable on stable.